### PR TITLE
Hello Microverse Project

### PR DESCRIPTION
**Pull request by myaserkhan**
The following files and changes committed:

added an html and css files
added test.md and node_modules file and added it to .gitignore
updated the template README md file
added the hint and stylint linter frameworks

Thanks for the review 😃

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# This file should be ignored by git
+test.md
```
Comment (cynthiainga) [inline .gitignore]: - [x] Please kindly add the `node_modules` to your `.gitignore file` ☝️ ☝️

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @MYaserKhan, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- [ ] No linters errors ✔️ 
- [ ] Good commit history ✔️ 
- [ ] Used correct Github flow ✔️ 

## Required Changes ♻️

- [ ] Please kindly make sure your README file is more descriptive, you need to customize the README file and add the necessary information 🙃 🙃 
- [ ] Please the Pull Request should have a descriptive title telling you what changes are inside for example `Hello Microverse Project` 😁 😁
- [ ] I think it will be better if the Pull Request has a summary describing introduced changes in more detail so that your PR will look more professional  😎 😎
- [ ] There is a .gitignore file listing `test.md` file and `node_modules`, please kindly add the `node_modules` to your `.gitignore file` 🙃 🙃 
- [ ] Please kindly setup correctly the linters, add the [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) and [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) files to your repo. You can use [this link](https://github.com/microverseinc/linters-config/tree/master/html-css) to have more details 🙃 🙃 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (myaserkhan) [reply .gitignore]: I have made the necessary changes required in the project repo. @cynthiainga please share your feedback. 
Thanks

**Review by mwanawabangona — APPROVED** [review]
Hi @ ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations!  


## Highlights
*   No linter errors :heavy_check_mark:
*   Repo looks professional :briefcase:
*   Solid pull request scroll
*   Good quality commits
*   Good job implementing the requested changes!! :clap: :clap: :clap:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**